pkg/webhook/clusterpropagationpolicy/mutating.go

```diff
@@ -66,10 +64,6 @@ func (a *MutatingAdmission) Handle(_ context.Context, req admission.Request) adm
 	helper.AddTolerations(&policy.Spec.Placement, helper.NewNotReadyToleration(a.DefaultNotReadyTolerationSeconds),
 		helper.NewUnreachableToleration(a.DefaultUnreachableTolerationSeconds))
 
-	if len(policy.Name) > validation.LabelValueMaxLength {
```

Comment:
Just one question: Do we have e2e to test this case now?

The length of pp's name exceeds 63.

Reply:
name is so long, it may not be a common test case.

Reply:
I suggest adding a case for testing.

Reply:
Ok~

Reply:
Update, please help a review again.